browser/about_flags.cc

```diff
@@ -406,6 +408,10 @@ constexpr char kBraveBackgroundVideoPlaybackDescription[] =
     "Enables play audio from video in background when tab is not active or "
     "device screen is turned off. Try to switch to desktop mode if this "
     "feature is not working.";
+constexpr char kBraveAndroidSafeBrowsingName[] = "Safe Browsing";
+constexpr char kBraveAndroidSafeBrowsingDescription[] =
+    "Enables safe browsing for determining whether a URL has been marked as a "
```

Comment:
I would say "Google Safe Browsing" instead of "safe browsing".

Reply:
I changed it to `Google Safe Browsing`